system/trace/trace_dump.c

```diff
@@ -616,6 +632,34 @@ static int trace_dump_one(trace_dump_t type, FAR FILE *out, FAR uint8_t *p,
         break;
 #endif
 
+#ifdef CONFIG_SCHED_INSTRUMENTATION_CSECTION
+      case NOTE_CSECTION_ENTER:
+      case NOTE_CSECTION_LEAVE:
+        {
+          struct note_csection_s *ncs;
+          ncs = (FAR struct note_csection_s *)p;
+          trace_dump_header(out, &ncs->ncs_cmn, ctx);
+          fprintf(out, "tracing_mark_write: %c|%d|critical_section\n",
```

Comment:
should we output the linux kernel format too?

Reply:
This is compatible with the linux kernel format.
In the linux kernel, tracing_mark_write is to mark some user data, adding this content will not affect the analysis.
At the same time, I added tracing_mark_write to both dump_string and dump_binary for consistency